wormhole-connect/src/hooks/useTrackTransfer.ts

```diff
@@ -60,9 +69,25 @@ const useTrackTransfer = (): void => {
         } catch (e) {
           console.error('Error tracking transfer:', e);
```

Comment:
My only concern is if the `sleepTime` is near 1 second and we are getting rate limited and keep on hitting the RPC. Should we backoff if an exception is thrown? That might complicate this, though. Just wanted to raise this point